internal/payloads/reservation_payload.go

```diff
@@ -134,6 +134,9 @@ type GCPReservationResponsePayload struct {
 
 	// Immediately power off the system after initialization
 	PowerOff bool `json:"poweroff"`
+
+	// Instances IDs, only present for finished reservations.
+	Instances []string `json:"instances,omitempty"`
```

Comment:
I will make sense to create struct here instead of string. It is very likely that we will want to also fetch DNS and IP of the launched instances alongside id and refactoring later is painful, as you can see here: https://github.com/RHEnVision/provisioning-backend/pull/448/files#diff-93db714a65f393cc33692e57cab38ad7ae1b8b12b3a0cefb1769b435b747a0d1

For now, I suggest to create a struct type with just a string as the single field so we can add IP/DNS later.

Reply:
I see that Amir has added `InstanceResponse` type for AWS. 
I guess I should wait until his PR will be merged?